openslides_backend/action/motion_submitter/create.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Dict
+
+from ...models.models import MotionSubmitter
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ..action import register_action
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
+
+
+@register_action("motion_submitter.create")
+class MotionSubmitterCreateAction(CreateAction):
+    """
+    Action to create a motion submitter.
+    """
+
+    model = MotionSubmitter()
+    schema = DefaultSchema(MotionSubmitter()).get_create_schema(
+        properties=["motion_id", "user_id"],
+        required_properties=["motion_id", "user_id"],
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        check if motion and user have the same meeting_id
+        """
+        motion_meeting_id = self.database.get(
+            FullQualifiedId(Collection("motion"), instance["motion_id"]), ["meeting_id"]
+        ).get("meeting_id")
+        user_meeting_id = self.database.get(
```

Comment:
Just an FYI: user don't generally belong to a meeting (only if they are temporary), so don't assume that a user has a meeting_id. I seem to have missed this when merging this.